### PR TITLE
Additional Tests for CDN/EventHub/Traffic Manager Profile

### PR DESCRIPTION
**Pull request by lrxtom2**
5 files changed and 11 cases added.
resource_arm_cdn_profile_test.go
resource_arm_eventhub_consumer_group_test.go
resource_arm_eventhub_namespace_test.go
resource_arm_eventhub_test.go
resource_arm_traffic_manager_profile_test.go

### CODE REVIEW

File: azurerm/resource_arm_eventhub_namespace_test.go [hunk 1]
```diff
@@ -241,3 +382,78 @@ resource "azurerm_eventhub_namespace" "test" {
...
+  name     = "acctestRG-%d"
+  location = "%s"
+}
+
+resource "azurerm_eventhub_namespace" "test" {
+  name                     = "acctesteventhubnamespace-%d"
+  location                 = "${azurerm_resource_group.test.location}"
+  resource_group_name      = "${azurerm_resource_group.test.name}"
+  sku                      = "Basic"
+  capacity                 = "2"
```
Comment (tombuildsstuff) [inline azurerm/resource_arm_eventhub_namespace_test.go]: can we remove the quotes around the integer here?

File: azurerm/resource_arm_eventhub_namespace_test.go [hunk 2]
```diff
@@ -241,3 +382,78 @@ resource "azurerm_eventhub_namespace" "test" {
...
+  name     = "acctestRG-%d"
+  location = "%s"
+}
+
+resource "azurerm_eventhub_namespace" "test" {
+  name                     = "acctesteventhubnamespace-%d"
+  location                 = "${azurerm_resource_group.test.location}"
+  resource_group_name      = "${azurerm_resource_group.test.name}"
+  sku                      = "Basic"
+  capacity                 = "20"
```
Comment (tombuildsstuff) [inline azurerm/resource_arm_eventhub_namespace_test.go]: can we remove the quotes around the capacity here?

File: azurerm/resource_arm_eventhub_namespace_test.go [hunk 3]
```diff
@@ -241,3 +382,78 @@ resource "azurerm_eventhub_namespace" "test" {
...
+}
+
+resource "azurerm_eventhub_namespace" "test" {
+  name                = "acctesteventhubnamespace-%d"
+  location            = "${azurerm_resource_group.test.location}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+  sku                 = "Basic"
+  tags {
+	environment = "Production"
+  	}
```
[the remark this answers is not in the document]
Comment (jeffreyCline) [reply azurerm/resource_arm_eventhub_namespace_test.go]: @lrxtom2 This got me too for a while, the `make fmt` will fix the spacing for the code, but not the mock strings in config functions. Please update as @tombuildsstuff has suggested.

**Comment by lrxtom2** [comment]
@tombuildsstuff  Thanks for review, I have submitted a commit to fix this comments.

File: azurerm/resource_arm_eventhub_namespace_test.go [hunk 4]
```diff
@@ -241,3 +382,78 @@ resource "azurerm_eventhub_namespace" "test" {
...
+  name                     = "acctesteventhubnamespace-%d"
+  location                 = "${azurerm_resource_group.test.location}"
+  resource_group_name      = "${azurerm_resource_group.test.name}"
+  sku                      = "Basic"
+  capacity                 = 20
+}
+`, rInt, location, rInt)
+}
+
+func testAccAzureRMEventHubNamespace_basicWithCapacityUpdate(rInt int, location string) string {
```
Comment (JunyiYi) [inline azurerm/resource_arm_eventhub_namespace_test.go]: Refactoring suggestion. You can also combine this with `testAccAzureRMEventHubNamespace_basicWithCapacity`. You just need to introduce a parameter called `capacity`.

**Pull request merged**